packages/@cdktf/provider-generator/lib/get/__tests__/generator/provider.test.ts

```diff
@@ -47,3 +47,5 @@ test("generate provider with only block_types", async () => {
   );
   expect(output).toMatchSnapshot();
 });
+
+test("name in constraint does not match resolved name in fqpn", () => {});
```

Comment:
In TDD, when they say start with a test, they mean the entire test with assertions and so on 🤣  :trollface:

Reply:
Oh wow not sure how that slipped by me 😂